### ISSUE TITLE
Custom ticket statuses

### ISSUE THREAD

**Issue by acoolov** [YetiForceCompany/YetiForceCRM#11837]
Hi all!

How can I create custom ticket statuses and replace existing ones? 
Would anyone be able to explain step by step how to accomplish it correctly so that my new ticket status properly closes the ticket. I assume I need to create a new pick list, name it Status. How should I do it correctly? Also so that the new Closed is going to mean the ticket is closed.

**Comment by vovpff** [comment]
You can try anything here: https://gitdeveloper.yetiforce.com/

P.S. You can add unlimited amount of picklist values

**Comment by acoolov** [comment]
Vladimir, do you know why I cannot edit existing ticket status values? It won't let me.

**Comment by vovpff** [comment]
@acoolov because a lot of system stuff depends on tickets status. You only can add and edit selfmade values.

**Comment by acoolov** [comment]
Oh ok. So i won't be able to just have Open, In Progress, Pending, and Closed ticket statuses? If i remove existing ticket status field from the ticket, the ticket module won't even load. Gives me error.

How can i just have those four ticket statuses?

Thanks a lot for your help.

**Comment by acoolov** [comment]
So there is no way in YetiForce to have custom ticket statuses working?

**Comment by acoolov** [comment]
How can I only have these four ticket statuses, and not a bunch of existing ones that completely don't fit our environment? I only need these four:

Open
In Progress
Pending
Closed

I can't seem to make it work. Once I remove existing ticket status field, the ticket module won't even load anymore.

**Comment by acoolov** [comment]
Does anyone know how to do this? I was really hoping we can start using this software for our environment, but with these ticket statuses, it won't be possible.

**Comment by acoolov** [comment]
@mariuszkrzaczkowski 

Do you happen to know how to do it?

**Comment by mariuszkrzaczkowski** [comment]
duplicate #11838
https://github.com/YetiForceCompany/YetiForceCRM/issues/11838#issuecomment-546426381

**Comment by acoolov** [comment]
Was able to accomplish this by re-installing Yetiforce and meeting all requirements first.
Then went to modify language variable for some ticket status values.
Then disabled the ones I didn't need in pick list for tickets.
Now I have only ticket statuses that I need without unnecessary statuses.

**Comment by vovpff** [comment]
My congrats!
